### PR TITLE
print username used by SP connector for NTLM 

### PR DESCRIPTION
**Pull request by sergres**
print process owner username in case of fallback to NTLM authentication (windows integrated
authentication).

### CODE REVIEW

**Review by lavanyachandramouli — APPROVED** [review]
LGTM.

File: src/com/google/enterprise/adaptor/sharepoint/SharePointFormsAuthenticationHandler.java [hunk 1]
```diff
@@ -27,6 +27,7 @@
 import javax.xml.ws.BindingProvider;
 import javax.xml.ws.WebServiceException;
 import javax.xml.ws.handler.MessageContext;
+import com.google.enterprise.adaptor.sharepoint.SharePointAdaptor;
```
Comment (wiarlawd) [inline src/com/google/enterprise/adaptor/sharepoint/SharePointFormsAuthenticationHandler.java]: This goes first, not last. Also, a static import is probably better, and those go first in a separate block before the other imports.

File: src/com/google/enterprise/adaptor/sharepoint/SharePointFormsAuthenticationHandler.java [hunk 2]
```diff
@@ -98,8 +100,9 @@ public AuthenticationResult authenticate() throws IOException {
     if (result.getErrorCode() != LoginErrorCode.NO_ERROR) {
       log.log(Level.WARNING, "Forms authentication failed with error code {0}. "
           + "Possible SharePoint environment with multiple claims providers. "
-          + "Adaptor will fallback to use windows integrated authentication.",
-          result.getErrorCode());      
+          + "Adaptor will fallback to use windows integrated authentication."
+          + "using username \"{1}\"", new Object[] { result.getErrorCode(),
+          SharePointAdaptor.getAdaptorUser("")});
```
Comment (wiarlawd) [inline src/com/google/enterprise/adaptor/sharepoint/SharePointFormsAuthenticationHandler.java]: Either with spaces inside braces or without is fine, but not a mix. Once getAdaptorUser is statically imported and the whole array fits on one line, break before new Object[] for a higher-level line break.

**Review by wiarlawd — CHANGES_REQUESTED** [review]
Formatting nits. Please amend the commit (git commit --amend) and force push (git push -f).

John L

File: src/com/google/enterprise/adaptor/sharepoint/SharePointFormsAuthenticationHandler.java [hunk 3]
```diff
@@ -98,8 +101,9 @@ public AuthenticationResult authenticate() throws IOException {
     if (result.getErrorCode() != LoginErrorCode.NO_ERROR) {
       log.log(Level.WARNING, "Forms authentication failed with error code {0}. "
           + "Possible SharePoint environment with multiple claims providers. "
-          + "Adaptor will fallback to use windows integrated authentication.",
-          result.getErrorCode());      
+          + "Adaptor will fallback to use windows integrated authentication."
```
Comment (wiarlawd) [inline src/com/google/enterprise/adaptor/sharepoint/SharePointFormsAuthenticationHandler.java]: Change the final . to space, as you did above.

**Review by wiarlawd — CHANGES_REQUESTED** [review]
One more nit. It's user-visible or I might have let it go.

John L

**Review by wiarlawd — APPROVED** [review]

**Pull request merged**